docs/RulesDescription.md

```diff
@@ -18,6 +18,8 @@ ButtonShouldHavePatterns | A button must support one of these patterns: Invoke,
 ButtonInvokeAndTogglePatterns | A button must not support both the Invoke and Toggle patterns. | WCAG 4.1.2 NameRoleValue | Error
 ButtonInvokeAndExpandCollapsePatterns | A button may have the Invoke and ExpandCollapse patterns together, but it is not recommended. If possible, please have only one of them.  | WCAG 1.3.1 InfoAndRelationships | Warning
 ButtonToggleAndExpandCollapsePatterns | A button must not support both the Toggle and ExpandCollapse patterns. | WCAG 4.1.2 NameRoleValue | Warning
+ClickablePointOnScreen  | An element's IsOffScreen property must be false when its clickable point is on screen. | Section 508 502.3.1 ObjectInformation | Warning
```

Comment:
@ferbonnin, is this a manual edit? It's different from the auto-generated file at https://github.com/microsoft/axe-windows/releases/download/v0.9.0/axe-windows-rules-0.9.0.md

Reply:
yes. all this file is manual edits. the auto-generated file doesn't have whether the issue is a warning or a failure